Tests/installation_tests/shared_unit_tests/StripeIdentity/StripeIdentityAssetTests.swift

```diff
@@ -7,7 +7,7 @@
 import UIKit
 import XCTest
 
-@testable import StripeIdentity
```

Comment:
it seems Tests/installation_tests/shared_unit_tests/StripeCore is also using `@test`, should we update those as well?

Reply:
Those tests appear to be unused, good catch. We should probably include them in the install tests.